### ISSUE TITLE
About metric

### ISSUE THREAD

**Issue by imlixinyang** [elvisyjlin/RelGAN-PyTorch#1]
Hi, can you share the script to reproduce your great metrics in your paper. Also, i want to compare my own model with yours, so it will help a lot if you can release more details! Thank you anyway!

**Comment by willylulu** [comment]
Thank you for your issue.

This repo hasn't included pretrain model yet.
We will update the pretrain model soon.

Check out this repo https://github.com/willylulu/RelGAN
If you want to reproduce the metrics identically, use the pretrain model in this repo (generator519.h5)
We calculated matrics in the paper with this model.

I think the experiment written in the paper is clearly understood. So we only provide the simplest script in this repo. Clone it and modify this script you will easily get the data in the paper.

If you have any question about the detail of the experiment, contact us.

**Comment by imlixinyang** [comment]
Some details about the metrics:
1. Acc and SSIM, did you just translate every label in every image in test dataset and calculate each for averange?
2. FID, how to get the folder based on test dataset to calculate fid?
Thank you a lot,

**Comment by elvisyjlin** [comment]
Yes for 1.

For 2., our method is a little bit tricky. We first create a "target attribute list". We randomly permute the 30k CelebA-HQ attributes twice to get the target attribute list with 60k attributes. And we translate the faces according to the target attribute list so that the generated faces are supposed to fit the distribution of CelebA-HQ. Then we calculate the FID score from the feature statistics of 30k CelebA-HQ faces and that of 60k generated faces for each method (StarGAN, AttGAN, and RelGAN).

I'm sorry for our late reply. I'm currently doing my military service so I can barely use my laptop on weekdays.

**Comment by imlixinyang** [comment]
That's okay, it's already been very kind of you to response these details.
Thank you again!

**Comment by mxkai** [comment]
Hi, I have a question about CelebA-HQ, how does the dataset divide the training set and the test set

**Comment by mxkai** [comment]
Hi, I have a question about Celeba-HQ. Can you tell me how the dataset divides the training set and the test set
